src/test/java/io/aiven/kafka/auth/AivenAclAuthorizerTest.java

```diff
@@ -65,11 +74,76 @@ public void testAivenAclAuthorizer() throws IOException {
         auth.configure(configs);
 
         // basic ACL checks
-        assertTrue(auth.checkAcl("User", "pass", "Read", "Topic:Target"));
-        assertFalse(auth.checkAcl("User", "fail", "Read", "Topic:Target"));
-        assertFalse(auth.checkAcl("User", "pass", "Read", "Fail:Target"));
-        assertFalse(auth.checkAcl("User", "pass", "FailRead", "Topic:Target"));
-        assertFalse(auth.checkAcl("NonUser", "pass", "Read", "Topic:Target"));
+        assertTrue(
+            auth.authorize(
+                new Session(
+                    new KafkaPrincipal("User", "pass"),
+                    InetAddress.getLocalHost()
+                ),
+                Operation.fromJava(AclOperation.READ),
+                new Resource(
+                    ResourceType.fromJava(org.apache.kafka.common.resource.ResourceType.TOPIC),
```

Comment:
I'd import `org.apache.kafka.common.resource.ResourceType` like others from than package.

Reply:
class ResourceType has same name for diff packages :-(

Reply:
No worries